rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -230,8 +222,7 @@ JobInfoData jobInfo(
     @GZIP
     @Path("jobs/{jobid}/result/value")
     @Produces("application/json")
-    Map<String, String> jobResultValue(
-            @HeaderParam("sessionid") String sessionId,
+    Map<String, String> jobResultValue(@HeaderParam("sessionid") String sessionId,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: org.ow2.proactive_grid_cloud_portal.scheduler.exception.NotConnectedRestException, org.ow2.proactive_grid_cloud_portal.scheduler.exception.PermissionRestException, org.ow2.proactive_grid_cloud_portal.scheduler.exception.UnknownJobRestException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1160)